vtex-apps/rebuy
# Add basic rebuy functionality

#### What is the purpose of this pull request?
Create the rebuy app with basic functionality.

*NOTE*: It's using fetch to query MasterData v2 documents. I've started work at `vtex.store-graphql` to enable MDv2 integration, but it will take some time to get it to work reliably and not break MDv1 queries and mutations.

#### What problem is this solving?
Increase recurrency for repeat orders.

#### How should this be manually tested?

1. Visit the [workspace](https://rebuy--delivery.myvtex.com/order)
2. Use the following phone number on "Redeem Address" which already has an order: `48987654321`
2.a. OR do the full purchase flow to create an order, go back to the landing page with address locator and then use the phone number with which you completed the order with
3. `vtex.rebuy` is shown right after the carousel in this case
4. Add the products to cart and it should open and show the same products that are on rebuy

#### Screenshots or example usage
![screenshot at oct 31 17-57-30](https://user-images.githubusercontent.com/27775611/47818282-7ecaea00-dd36-11e8-9ef6-9f19da4aafac.png)

#### Types of changes
